crates/proof-of-sql/src/sql/proof/query_proof.rs

```diff
@@ -46,37 +45,51 @@ fn get_index_range<'a>(
         .unwrap_or((0, 1))
 }
 
-/// The proof for a query.
-///
-/// Note: Because the class is deserialized from untrusted data, it
-/// cannot maintain any invariant on its data members; hence, they are
-/// all public so as to allow for easy manipulation for testing.
 #[derive(Clone, Serialize, Deserialize)]
-pub(super) struct QueryProof<CP: CommitmentEvaluationProof> {
-    /// Bit distributions
-    pub bit_distributions: Vec<BitDistribution>,
+pub struct QueryProofMessage1<C> {
```

Comment:
Maybe we should use `QueryProofMessageFirstRound` and `QueryProofMessageFinalRound` instead?

Reply:
Good idea.